lib/gs-plugin-loader.c

```diff
@@ -2501,23 +2486,16 @@ mount_added_cb (GVolumeMonitor *volume_monitor,
 {
 	GsPluginLoaderPrivate *priv = gs_plugin_loader_get_instance_private (plugin_loader);
 	g_autoptr(GDrive) drive = g_mount_get_drive (mount);
-	g_autoptr(GMount) existing_mount = NULL;
 
 	if (!g_drive_is_removable (drive))
 		return;
 
-	existing_mount = find_mount_in_list (priv->removable_mounts, mount);
-
-	/* update the list to remove any equivalent mounts and move this to the
-	 * front so we prioritize mounts by recency
+	/* update the list to remove this mount if it already exists mounts and
```

Comment:
`if it already exists mounts`
Maybe there's a comma missing or an extra word here.

Reply:
Fixed locally